.github/workflows/pkgdown.yaml

```diff
@@ -1,50 +1,70 @@
 on:
   push:
-    branches: master
+    branches: quillt
+  workflow_run:
+    workflows: ["R-CMD-check"]
+    branches: [main,master]
+    types: completed
```

Comment:
```suggestion
```

This should be removed. See last edition to the worflow I have made in https://github.com/rstudio/rmarkdown/blob/master/.github/workflows/pkgdown.yaml